tasks/copy_files_to_archive/test/unit_tests/ConfigCheck.py

```diff
@@ -13,4 +13,4 @@ def check_multipart_chunksize(
             if Config.multipart_chunksize != self.multipart_chunksize:
                 self.bad_config = Config
         except Exception as ex:
-            bad_config = ex
+            bad_config = ex  # noqa
```

Comment:
Just out of curiosity, do you remember what Flake8 didn't like here?

Reply:
`local variable bad_config is assigned to but never used`

I think I can just do this to remove the error:

```
except Exception as ex:
   ex
```

Reply:
Ah. Try `self.bad_config = ex`
I might have messed up my scope when storing `ex`.